src/main/java/com/training/springpractice/AppConfig.java

```diff
@@ -0,0 +1,15 @@
+package com.training.springpractice;
```

Comment:
As a suggestion, when you need to add configuration files, in order to mantain an order in the structure of the project it is a good practice to define a package where you will centralize all Configurations files you will need.  In that way the Configuration files are in a single place.